funnel/views/account.py

```diff
@@ -491,10 +506,10 @@ def change_password():
     return render_form(
         form=form,
         title=_("Change password"),
-        formid='changepassword',
+        formid='password-change',
         submit=_("Change password"),
-        cancel_url=url_for('account'),
```

Comment:
User should be able to cancel this operation.

Reply:
Mobile layout has an X button. If web layout also does, this is fine.

Reply:
Add X to web layout